### ISSUE TITLE
Set style for FocusZone focus outline?

### ISSUE THREAD

**Issue by gscottolson** [OfficeDev/office-ui-fabric-react#3913]
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

#### Describe the issue:

The FocusZone focus outline is thin, solid black border. Is there any way to enhance it?

#### If applicable, please provide a codepen repro:

You can see the behavior here: https://developer.microsoft.com/en-us/fabric#/components/focuszone

**Comment by gscottolson** [comment]
I found the reference in the docs that pointed me to the [associated SCSS mixin](https://github.com/OfficeDev/office-ui-fabric-react/blob/3ecbbe49168a0dc4524304825d08ff0ab7fc4ccd/packages/example-app-base/src/common/_focusBorder.scss). Just wanted to update this issue in case others were looking for the same thing.

**Comment by Jahnp** [comment]
Just to be clear, the FocusZone itself doesn't mandate any focus styling out of the box--as you point out that's handled by the `focusBorder` mixin for SCSS-based components. That said, what scenario are you trying to support? Generally, focus borders should be as consistent as possible within product families and should only be adjusted if absolutely needed (e.g. for accessibility or to stand out on backgrounds)

**Comment by jordandrako** [comment]
To expand on @Jahnp's comment, for components using the getStyles api there is a getFocus() function as well that takes a theme object as an argument, and returns focus border styles. Closing since this is answered.
